Practice#4

```diff
@@ -0,0 +1,22 @@
+import java.util.Comparator;
+import java.util.Set;
+
+public class TreeSet implements Comparator<String> {
+
+    @Override
+    public int compare(String o1, String o2) {
```

Comment:
& what about additional logic in case if string contains 1 symbol or for ""?
Erin, all these tasks should be done properly - believe me - I have no possibility to check & add comments to all not finished solutions